integration_tests/.gitignore

```diff
@@ -0,0 +1 @@
+*.txt
```

Comment:
are we not generating these in the temp_dir already? why do we need to ignore them?